### PR TITLE
DOC-5481: Added migration procedure from saslauthd to native ldap

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/manage/pages/manage-security/configure-saslauthd.adoc [hunk 1]
```diff
@@ -19,11 +19,46 @@ The `saslauthd` agent, which must be installed and configured on each Couchbase
...
+
+. Configure Native LDAP.
+This is described in detail, in xref:manage:manage-security/configure-ldap.adoc[Configure LDAP.]
++
+Note that the procedure provides two principal options, which are *Enable LDAP user authentication* and *Enable LDAP group authentication & sync*.
+During your configuration procedure:
+
+** You _must_ select *Enable LDAP user authentication*, and then configure associated details.
+
+** If LDAP group authentication is required, you _can_ select *Enable LDAP group authentication & sync*, and then configure associated details.
```
Comment (timofey-barmin) [inline modules/manage/pages/manage-security/configure-saslauthd.adoc]: "LDAP group authentication" -> "LDAP group authorization"

File: modules/manage/pages/manage-security/configure-saslauthd.adoc [hunk 2]
```diff
@@ -19,11 +19,46 @@ The `saslauthd` agent, which must be installed and configured on each Couchbase
...
+Alternatively, you can leave this option disabled for now: it does not need to have been enabled, for migration to succeed.
+Instead, it can be enabled and configured later, after migration has been completed.
+
+. Disable `saslauthd` authentication, as follows:
++
+----
+couchbase-cli setting-saslauthd -c 127.0.0.1:9001 -u Administrator -p password --enabled 0
```
Comment (timofey-barmin) [inline modules/manage/pages/manage-security/configure-saslauthd.adoc]: please replace "-c 127.0.0.1:9001" with something that usually is used in examples. I see that "-c 10.143.192.101" is used in example on the same page.

**Review by timofey-barmin — CHANGES_REQUESTED** [review]
Please also add link to this piece from configure-ldap, so people who start configuring LDAP can see that they need to migrate if they are already using saslauthd.

File: modules/manage/pages/manage-security/configure-saslauthd.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/manage/pages/manage-security/configure-saslauthd.adoc]: Done.

File: modules/manage/pages/manage-security/configure-saslauthd.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/manage/pages/manage-security/configure-saslauthd.adoc]: Done, thanks.

File: modules/manage/pages/manage-security/configure-saslauthd.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/manage/pages/manage-security/configure-saslauthd.adoc]: I've also added a section and link as /manage/manage-security/configure-ldap.html#migrating-to-native-ldap, and have added a link to /learn/security/authentication-overview.html#saslauthd-and-pam

File: modules/manage/pages/manage-security/configure-saslauthd.adoc [hunk 3]
```diff
@@ -19,11 +19,46 @@ The `saslauthd` agent, which must be installed and configured on each Couchbase
...
+
+. Configure Native LDAP.
+This is described in detail, in xref:manage:manage-security/configure-ldap.adoc[Configure LDAP.]
++
+Note that the procedure provides two principal options, which are *Enable LDAP user authentication* and *Enable LDAP group authorization & sync*.
+During your configuration procedure:
+
+** You _must_ select *Enable LDAP user authentication*, and then configure associated details.
+
+** If LDAP group authentication is required, you _can_ select *Enable LDAP group authorization & sync*, and then configure associated details.
```
Comment (timofey-barmin) [inline modules/manage/pages/manage-security/configure-saslauthd.adoc]:
```suggestion
** If LDAP group authorization is required, you _can_ select *Enable LDAP group authorization & sync*, and then configure associated details.
```

**Review by timofey-barmin — CHANGES_REQUESTED** [review]

**Review by timofey-barmin — APPROVED** [review]

**Pull request merged**